Sources/_OpenAPIGeneratorCore/Translator/Responses/translateResponse.swift

```diff
@@ -20,6 +20,7 @@ extension TypesFileTranslator {
     ///   - typedResponse: The typed response to declare.
     /// - Returns: A structure declaration.
     func translateResponseInTypes(
+        responseKind: String = "",
```

Comment:
```suggestion
        responseKind: ResponseKind,
```

As below, let's pass in the typesafe variant and not provide a default.